### DISCUSSION TITLE
Is the Whisper-Timeout even with the callback function 20 minutes?

### DISCUSSION THREAD

**Discussion by iKushiu** [deepgram/community#537] [General help] [answered]
My question is if the timeout for whispers is also 20 minutes if I use the transcribe_url_callback() function.
Or can you use this function for really long audio files? (i talk about podcasts so like 6 hours or even more)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @iKushiu, the time for whisper is 20 minutes even when using a callback. We recommend breaking long audio files into 30-60 minute chunks.

**Reply by iKushiu** [reply]
Okay yeah thanks.
